test/0.multipleQueries.js

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2018-2019, NuoDB, Inc.
+// All rights reserved.
+//
+// Redistribution and use permitted under the terms of the 3-clause BSD license.
+
+'use strict';
+
+var { Driver } = require('..');
+var should = require('should');
+var async = require('async');
+var config = require('./config');
+var helper = require('./typeHelper');
+
+describe('14. typeDouble.js', function () {
```

Comment:
Should this be renamed 0. multipleQueries.js instead of 14 . typeDouble.js?

Reply:
This file was included in the commit accidentally. I was using this as a scratch pad while working. I've reset and removed the file.